betterproto/__init__.py

```diff
@@ -776,7 +757,7 @@ def FromString(cls: Type[T], data: bytes) -> T:
         return cls().parse(data)
 
     def to_dict(
-        self, casing: Casing = Casing.CAMEL, include_default_values: bool = False
+        self, casing: Optional[Casing] = None, include_default_values: bool = False
```

Comment:
Since Enums are not mutable, do we need this change? Or was passing a default not possible with the new class?

I think it was helpful for developers to see the default casing from the parameter.

Reply:
Unfortunately it raises an AttributeError as the class hasn't been created yet.

Reply:
Actually, I guess we don't need to use Betterproto Enums for Casing. Perhaps just keep things simple and use the python Enum instead?

Reply:
I don't think it's that big of a deal, if someone wants to see the default they can just view source.